### DISCUSSION TITLE
Oh The Biomes We've Gone Compatibility addon

### DISCUSSION THREAD

**Discussion by ENC4YP7ED** [DynamicTreesTeam/DynamicTrees#984] [Compatibility Addon Suggestions]
**The Original Mod (Oh The Biomes You'll Go)** Only goes up to Version **1.19.3** and nowerdays People use the Mod **Oh The Biomes We've Gone** for playing with **Oh The Biomes You'll Go** on newer Version so a Compatibility addon would be great to have since the Fork has much Downloads and would oftne be used with that Oh The Biomes We've Gone Compatibility addon I am asking about.

**Comment by ENC4YP7ED** [comment]
Fork : https://modrinth.com/mod/oh-the-biomes-weve-gone/versions
Original : https://modrinth.com/mod/biomesyougo/versions

**Comment by supermassimo** [comment]
Duplicate of https://github.com/orgs/DynamicTreesTeam/discussions/934
